json-definitions/v1/test-result/index.json

```diff
@@ -57,34 +90,61 @@
       }
     },
     "vehicleType": {
-      "type": "string"
+      "anyOf": [
+        {
+          "$ref": "../../v3/tech-record/enums/vehicleType.ignore.json"
+        }
+      ]
     },
     "vehicleConfiguration": {
       "type": "string"
     },
     "odometerReading": {
-      "type": ["number", "null"]
+      "type": [
+        "number",
+        "null"
+      ]
     },
     "odometerReadingUnits": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
```

Comment:
There should only be 2 options for this and as such an enum?

['kilometres', 'miles']

Reply:
I was being lazy, sorry